src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js

```diff
@@ -0,0 +1,59 @@
+const { getDrupalValue } = require('./helpers');
+
+function createMetaTag(type, key, value) {
+  return {
+    __typename: type,
+    key,
+    value,
+  };
+}
+
+const transform = ({
+  title,
+  path,
+  moderationState,
+  metatag: { value: metaTags },
+  fieldNicknameForThisFacility,
+}) => ({
+  entity: {
+    entityType: 'node',
+    entityBundle: 'health_care_region_page',
+    entityPublished: getDrupalValue(moderationState) === 'published',
+    entityLabel: getDrupalValue(title),
+    title: getDrupalValue(title),
+    entityUrl: {
+      path: path[0].alias.replace(/\\/g, ''),
```

Comment:
Is this needed? I _think_ when we parse the JSON, it'll read `\/` as a `/`, so we shouldn't need to strip them out.

Might be worth double-checking.